json_annotation/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 3.0.2
```

Comment:
If we decide to go forward with this feature it would need to be version `3.1.0`